docs/developer-docs/developer-welcome.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Welcome to the Dev Team
+description: Welcomes newly graduated developers, teaches how to upload a set, and educates on all of the privileges exclusive to developers.
+---
+
+# Welcome to the Dev Team!
+
+Now that you’re a dev I'm sure the first thing you want to know is how you can upload your set and what other options and privileges are now open to you. You'll have plenty of fancy new features on the website that you'll notice when going to a game’s page as well as a few other places, which we will go over here.
+
+[[toc]]
+
+## Set Management
+
+The web team is working on a new Management page which is where all dev related activities will end up being moved to. It can be accessed through the “Manage” button right above the Dev dropdown menu.
+
+This page must be used for editing release dates as well as **drag-and-drop reordering**.

```

Comment:
```suggestion
This page is used for editing release dates as well as **drag-and-drop reordering**.
```
One thing to note: eventually _all_ of the game metadata editing will be on the page (it will be completely removed from the normal site). May just be worth a mental note for now.